tests/integration/tier0/destructive/offline-system-conversion/prepare_system.py

```diff
@@ -1,31 +1,36 @@
+import os
 import re
 import socket
 
-from conftest import SATELLITE_PKG_DST, SATELLITE_PKG_URL, SATELLITE_URL, SYSTEM_RELEASE_ENV, TEST_VARS
+from conftest import TEST_VARS
 
 
-# Replace urls in rhsm.conf file to the satellite server
-# Without doing this we get obsolete dogfood server as source of repositories
 def replace_urls_rhsm():
+    """
+    Replace urls in rhsm.conf file to the satellite server
+    Without doing this we get obsolete dogfood server as source of repositories
+    """
```

Comment:
Maybe we will not need this function anymore since we do not download the katello package

Reply:
Good point, let's try that